frontend/src/slices/auth/auth.slice.ts

```diff
@@ -23,26 +25,32 @@ const { reducer, actions, name } = createSlice({
   extraReducers(builder) {
     builder.addCase(signUp.pending, (state) => {
       state.authenticatedUserDataStatus = DataStatus.PENDING;
```

Comment:
remove authenticatedUserDataStatus change for signUp/signIn

Reply:
oh, I got it)